qutebrowser/commands/command.py

```diff
@@ -515,3 +515,7 @@ def validate_mode(self, mode):
             raise cmdexc.PrerequisitesError(
                 "{}: This command is only allowed in {} mode, not {}.".format(
                     self.name, mode_names, mode.name))
+
+    def takes_count(self):
+        """Return true iff this command can take a count argument."""
+        return any(arg.count for arg in self._qute_args)
```

Comment:
Another blast from the past (fixed in bfa518896a695f0545dedb708f4e6db5667a9dd8 together with some refactoring): This always returned `True` when a command had a `@cmdutils.argument` annotation - because what actually was going on here is that the *keys* of the `self._qute_args` dict got checked, which are strings, and strings have a `.count()` method. :wink: